### PR TITLE
Windows enable flamegraphs

### PR DESCRIPTION
**Pull request by marcin-ol**
Enable flamegraph functionality on windows.

## Description
Add script to build flamegraph tool - burn - on windows.

## Related Issue
#685 

## Screenshots
2023-02-16 18_55_58-gprofiler - Preview last_flamegraph html - Visual Studio Code

## Checklist:
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the relevant documentation.
- [ ] I have added tests for new logic.

### CODE REVIEW

File: scripts/windows/build.bat [hunk 1]
```diff
@@ -72,13 +84,18 @@ MKDIR granulate-utils 2> NUL
...
 python -m pip install --no-cache-dir -r exe-requirements.txt
 MKDIR gprofiler\resources\python
 COPY ..\py-spy\py-spy.exe gprofiler\resources\python
+COPY ..\burn\burn.exe gprofiler\resources
 
 ECHO D | XCOPY ..\gprofiler .\gprofiler /E
 FOR %%i in ("..\pyi_build.py" "..\pyinstaller.spec") do COPY "%%i" .
 pyinstaller pyinstaller.spec
+
+:clear_error
```
Comment (Jongy) [inline scripts/windows/build.bat:100]: Need to check if pyinstaller susceeds before `clear_error`?

File: scripts/windows/build.bat [hunk 2]
```diff
@@ -35,14 +35,14 @@ FOR /f "tokens=1-2" %%i in ('python --version') do (
...
-SET ERRORLEVEL=
+CALL :clear_error
 WHERE /Q pip
 IF ERRORLEVEL 1 (
         ECHO pip wasn't found. Attempting to install...
         wget -O .\dep\get-pip.py https://bootstrap.pypa.io/get-pip.py
         python  .\dep\get-pip.py
-	SET ERRORLEVEL=
-	WHERE /Q pip
+		CALL :clear_error
```
Comment (Jongy) [inline scripts/windows/build.bat:44]: Shouldn't this remain in the same indentation?

**Comment by Jongy** [comment]
Fix lint and Linux build :) Other than that, LGTM

File: scripts/windows/build.bat (hunk 1 above)
Comment (marcin-ol) [reply scripts/windows/build.bat:108]: Yes.

File: scripts/windows/build.bat (hunk 2 above)
Comment (marcin-ol) [reply scripts/windows/build.bat]: Indentation was mixed, thanks for catching. Will apply common formatting.

**Comment by Jongy** [comment]
Lint still fails

**Comment by Jongy** [comment]
Update from master and ensure Windows CI passes :)

**Comment by Jongy** [comment]
CI will pass now after merging from master.

**Review by Jongy — APPROVED** [review]

**Pull request merged**